openquake/sep/liquefaction/liquefaction.py

```diff
@@ -74,17 +76,52 @@
 
 
 FT_PER_M = 3.28084
+CM_PER_M = 100
 
 
-def zhu_magnitude_correction_factor(mag: float):
+def sigmoid(x):
+  return 1.0 / (1.0 + np.exp(-x))
+
+
+def _idriss_magnitude_scaling_factor(mag: float):
+    """
+    Youd, T. L., & Idriss, I. M. (2001).
+    Liquefaction Resistance of Soils: Summary Report from the 1996 NCEER and 1998 NCEER/NSF
+    Workshops on Evaluation of Liquefaction Resistance of Soils. Journal of Geotechnical
+    and Geoenvironmental Engineering, 127(4), 297–313.
+    https://doi.org/10.1061/(asce)1090-0241(2001)127:4(297)
+
+    """
+    return (10**2.24) / (mag**2.56)
```

Comment:
Since `10**2.24` is a constant it is a good practice to give it an all caps name and avoid recomputing it every time.